### PR TITLE
FT: compile using babel

### PR DESCRIPTION
**Pull request by rahulreddy**
Included commits

### ft: compile js to dist dir using babel
  As utapi is a module that will be required by other packages, the code
will be compiled by babel to the dist directory for distribution.

### 	ft: ignore dist folder
babel compiles from src to dist

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,6 +32,7 @@
     "mocha": "^3.0.2"
   },
   "scripts": {
+    "postinstall": "./node_modules/.bin/babel --ignore node_modules src/ --out-dir dist",
```
Comment (michael-zapata) [inline package.json]: You can just use `babel` instead, npm will infer that folder.

File: package.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
   "dependencies": {
     "arsenal": "scality/Arsenal#rel/6.2",
     "async": "^2.0.1",
+    "babel-cli": "^6.14.0",
```
Comment (michael-zapata) [inline package.json]: Could we freeze the version instead?

File: package.json (hunk 1 above)
Comment (rahulreddy) [inline package.json]: Ah, I wasn't sure if npm would pick that up.

File: package.json (hunk 2 above)
Comment (rahulreddy) [inline package.json]: Sure

**Comment by rahulreddy** [comment]
@michael-zapata I have addressed your comments.

**Pull request merged**